metadata-ingestion/src/datahub/ingestion/source/iceberg/iceberg_profiler.py

```diff
@@ -54,11 +63,14 @@ def _aggregate_bounds(
         manifest_values: Dict[int, Any],
     ) -> None:
         for field_id, value_encoded in manifest_values.items():  # type: int, Any
-            field: NestedField = schema.find_field(field_id)
-            # Bounds in manifests can reference historical field IDs that are not part of the current schema.
-            # We simply not profile those since we only care about the current snapshot.
-            if field and IcebergProfiler._is_numeric_type(field.type):
-                value_decoded = Conversions.from_byte_buffer(field.type, value_encoded)
+            try:
+                field = schema.find_field(field_id)
+            except ValueError:
+                # Bounds in manifests can reference historical field IDs that are not part of the current schema.
+                # We simply not profile those since we only care about the current snapshot.
+                continue
+            if IcebergProfiler._is_numeric_type(field.field_type):
+                value_decoded = from_bytes(field.field_type, value_encoded)
                 if value_decoded:
```

Comment:
```suggestion
                if value_decoded is not None:
```

This is a bug, it won't catch `0` and `0.0`, therefore the `passenger_count` and `trip_distance` is unknown:

![image](https://github.com/datahub-project/datahub/assets/1134248/71716568-55b3-439e-8156-c79340ee8327)

Reply:
oh wow...  I've noticed this behavior and never took the time to investigate.  Thank you for that! 
I'll make sure there is a test case for it.